src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -78,6 +78,10 @@ object SamResourceActions {
   val setPublic = ResourceAction("set_public")
   val readAuthDomain = ResourceAction("read_auth_domain")
   val testAnyActionAccess = ResourceAction("test_any_action_access")
+  val getParent = ResourceAction("get_parent")
```

Comment:
Outstanding question: Do I need to update reference.conf to add these actions to the various resource types listed there?

Reply:
not yet... we don't have any resource types that allow parents are children yet